2019-January-29/writingSamples/frida.tex

```diff
@@ -0,0 +1,141 @@
+In DS mice we evaluate effects of long-term ketogenic diet (KD)
+treatment on mortality and seizure frequency. DS mice showed high
+premature mortality (44\% survival by P60) that was associated with
+increased spontaneous GTCSs 1-2 days prior to death. KD treated mice had
+a significant reduction in mortality (86\% survival by P60) compared to
+mice fed a control diet. Interestingly, increased survival was not
+associated with decreased spontaneous non-fatal seizures. Further
+studies are needed to determine how KD confers protection from SUDEP.
+Moreover, our findings implicate time of day as a factor influencing the
+occurrence of seizures and SUDEP. DS mice, though nocturnal, are more
+likely to have SUDEP at night, suggesting that the increased incidence
+of SUDEP at night in DS mice may not be solely due to sleep.
+
+\textbf{INTRODUCTION}
+
+Sudden unexpected death in epilepsy (SUDEP) is estimated to occur in
+\textasciitilde{}17\% of patients with epilepsy\textsuperscript{1}. This
+number can drastically increase to 50\% in patients with poorly
+controlled and severe epilepsy\textsuperscript{2}. Although the
+mechanisms underlying SUDEP are not fully understood, an increasing body
+of evidence suggests SUDEP is due to seizure-induced cardiorespiratory
+dysfunction\textsuperscript{3,4}. However, little is known about the
+circumstances leading up to SUDEP. A strong association with sleep has
+been documented in a number of studies\textsuperscript{3,5}. Although a
+significant majority of patients are found in bed in the prone position
+at the time of death\textsuperscript{6-8}, the occurrence of SUDEP
+during sleep varies widely among published case
+studies\textsuperscript{9}. This suggests that other circadian factors
+may be relevant.
+
+Dravet Syndrome (DS) is a devastating epileptic encephalopathy of
+childhood-onset that typically manifests as febrile seizures in the
+first year of life and progresses to refractory
+epilepsy\textsuperscript{10}. Children with DS develop several
+comorbidities, such as ataxia, sleep disturbance and cognitive
+impairments\textsuperscript{11}. In patients with DS, the risk of SUDEP
+is estimated to be 15 times higher than in other pediatric epilepsies.
+Premature death occurs in 21\% of DS patients, with SUDEP accounting for
+nearly half of these deaths\textsuperscript{12}. Approximately 85\% of
+DS cases are caused by dominant loss-of-function mutations of the
+\emph{SCN1A} gene, which encodes the neuronal voltage-gated sodium
+channel Nav1.1\textsuperscript{11,13}. DS mouse models have proven to be
+an efficient research tool for understanding the pathophysiology of
+SUDEP as they recapitulate many aspects of the clinical condition: they
+have spontaneous seizures and a high incidence of premature
+mortality\textsuperscript{14}. They also display impaired sleep
+architecture homeostasis\textsuperscript{10}.
+
+A recent study found that time of day can have an independent influence
```

Comment:
I wonder if this result can be incorporated earlier in the introduction at the [end of the first paragraph](https://github.com/jdkent/writingWorkshop/pull/6/files#diff-8b0672475eee2da64b369ed980c6a95fR107)